frontend/src/pages/projects/notebook/DeleteNotebookModal.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { Alert, Button, Modal, Stack, StackItem } from '@patternfly/react-core';
+import { NotebookKind } from '../../../k8sTypes';
+import { getNotebookDisplayName } from '../utils';
+import { deleteNotebook } from '../../../api';
+
+type DeleteNotebookModalProps = {
+  notebook?: NotebookKind;
+  onClose: () => void;
```

Comment:
May we also set a boolean to see `submit` or not? If it's submitted then we refresh the notebooks, or there is no need to fetch all the notebooks.

Reply:
Fetching all notebooks isn't the end of the world as we don't have a periodic refresh -- but that's a fair point all the same. We could definitely make this better.